compiler-core/templates/gleam@@compile.erl

```diff
@@ -43,20 +60,71 @@ worker_loop(Parent, Out) ->
     Options = [report_errors, report_warnings, debug_info, {outdir, Out}],
     erlang:send(Parent, {work_please, self()}),
     receive
-        {module, Module} -> 
+        {module, Module} ->
             log({compiling, Module}),
             case compile:file(Module, Options) of
-                {ok, _} -> 
-                    log({compiled, Module});
-                error -> 
+                {ok, ModuleName} ->
+                    Beam = filename:join(Out, ModuleName) ++ ".beam",
+                    Message = {compiled, Beam},
+                    log(Message),
+                    erlang:send(Parent, Message);
+                error ->
                     log({failed, Module}),
                     erlang:send(Parent, failed)
             end,
             worker_loop(Parent, Out)
     end.
 
+compile_elixir(Modules, Out) ->
```

Comment:
Let's add a check to see if Elixir is installed before continuing, the error message willbe cryptic otherwise.